python/langsmith/client.py

```diff
@@ -4085,34 +4085,32 @@ def _prepare_multipart_data(
                 )
             )
 
-            if example.inputs:
-                inputsb = _dumps_json(example.inputs)
+            inputsb = _dumps_json(example.inputs)
```

Comment:
this we should probably do
```suggestion
            inputsb = _dumps_json(example.inputs or {})
```
same with outputs. to upload empty dict not null

Reply:
I think maybe ok to have `null` in example rows but don't feel strongly